python-scripts/src/main/python/dataproducts/services/misc/druid_job_submitter.py

```diff
@@ -41,12 +51,28 @@ def interpolate_config(self, report_config):
         print('String interpolation for the report config completed')
         return report_config_str
 
+    def check_schedule(self, reportSchedule,report_id):
+        if reportSchedule == 'DAILY' :
+          return True
+        elif reportSchedule == 'WEEKLY' :
+          if date.today().weekday() == 0:
+            return True
+        elif reportSchedule == 'MONTHLY' :
+          first_day = date.today().replace(day = 1)  
+          if date.today() == first_day :
+            return True
```

Comment:
Can this check be done like - `date.today().day == 1`?

Reply:
Yes. Updated